src/subsplit_dag.hpp

```diff
@@ -42,7 +67,7 @@ class SubsplitDAG {
   void PrintGPCSPIndexer() const;
   std::string ToDot(bool show_index_labels = true) const;
 
-  // Create a gpcsp indexer from ReversePostorderIndexTraversal and rootsplits_.
+  // Create a gpcsp indexer from ReversePostorderIndexTraversal.
```

Comment:
```suggestion
  // Create a GPCSPIndexer representing the DAG.
```

(Comments for public methods should describe intent, not implementation.)